core-web/libs/sdk/experiments/src/lib/dot-experiments.ts

```diff
@@ -0,0 +1,155 @@
+import { API_EXPERIMENTS_URL, EXPERIMENT_DB_KEY_PATH, EXPERIMENT_DB_STORE_NAME } from './constants';
+import { AssignedExperiments, IsUserIncludedApiResponse, SdkExperimentConfig } from './models';
+import { IndexDBDatabaseHandler } from './persistence/index-db-database-handler';
+import { dotLogger } from './utils/utils';
+
+/**
+ * `SdkExperiments` is a Typescript class to handles all operations related to fetching, storing, parsing, and navigating
```

Comment:
change `SdkExperiments` to `DotExperiments`

Reply:
done